### PR TITLE
587 remove frozen

### PR DESCRIPTION
**Pull request by jwestw**
## Pull Request submission

Removes the word frozen from the pipeline in both paths and in dictionary and Toml keys. 

#### Closes or fixes

* Detail the ticket(s) you are closing with this PR
RDRP-587

#### Code

- [x] **Code runs** The code runs on my machine and/or CDSW
- [x] **Conflicts resolved** There are no conflicts (I have performed a rebase if necessary)
- [ ] **Requirements** My/our code functions according to the requirements of the ticket
- [ ] **Dependencies** I have updated the environment yaml so it includes any new libraries I have used
- [ ] **Configuration file updated** any high level parameters that the user may interact with have been put into the config file (and imported to the script)
- [ ] **Clean Code**
    - [ ] Code is as [PEP 8]([url](https://peps.python.org/pep-0008/)) compliant as I can humanly make it
    - [ ] Code passess flake8 linting check
    - [ ] Code adheres to [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)
- [ ] **Type hints** All new functions have [type hints ](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html)

#### Documentation

Any new code includes all the following forms of documentation:

- [ ] **Function Documentation** Docstrings within the function(s')/methods have been created
    - [ ] Includes `Args` and `returns` for all major functions
    - [ ] The docstring details data types
- [ ] **Updated Documentation**: User and/or developer working doc has been updated

#### Data
- [ ] All data needed to run this script is available in Dev/Test
- [ ] All data is excluded from this pull request
- [ ] Secrets checker pre-commit passes

#### Testing
- [ ] **Unit tests** Unit tests have been created and are passing _or a new ticket to create tests has been created_

---

# Peer Review Section

- [ ] All requirements install from (updated) `environment.yaml`
- [ ] Documentation has been created and is clear - **check the working document**
- [ ] Doctrings ([Google format](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)) have been created and accurately describe the function's functionality
- [ ] Unit tests pass, or if not present _a new ticket to create tests has been created_
- [ ] **Code runs** The code runs on reviewer's machine and/or CDSW

#### Final approval (post-review)

The author has responded to my review and made changes to my satisfaction.
- [ ] **I recommend merging this request.**

---

### Review comments

*Insert detailed comments here!*

These might include, but not exclusively:

- bugs that need fixing (does it work as expected? and does it work with other code
  that it is likely to interact with?)
- alternative methods (could it be written more efficiently or with more clarity?)
- documentation improvements (does the documentation reflect how the code actually works?)
- additional tests that should be implemented (do the tests effectively assure that it
  works correctly?)
- code style improvements (could the code be written more clearly?)
- Do the changes represent a change in functionality so the version number should increase? Start a discussion if so.
- As a review you can generates the same outputs from running the code

Your suggestions should be tailored to the code that you are reviewing.
Be critical and clear, but not mean. Ask questions and set actions.

### CODE REVIEW

File: src/developer_config.yaml [hunk 1]
```diff
@@ -135,7 +137,7 @@ export_choices:
   export_intram_by_sic: "output_intram_by_sic_2023-12-06_v68.csv" # Can be set to None
   export_status_filtered: None # Can be set to None
   export_fte_total_qa: "total_fte_qa_2023-12-06_v68.csv" # Can be set to None
-  export_frozen_long_form: None # Can be set to None
+  export_long_form: None # Can be set to None
```
Comment (JenCheshire) [inline src/developer_config.yaml]: Tiny thing, but just for consistency, take out the underscore between long and form?

File: src/pipeline.py [hunk 2]
```diff
@@ -86,12 +86,30 @@ def run_pipeline(start, config_path):
     # Data Ingest
     MainLogger.info("Starting Data Ingest...")
 
+    # NI staging and validation
```
Comment (JenCheshire) [inline src/pipeline.py]: Is this supposed to be in this ticket?

File: src/outputs/gb_sas.py [hunk 3]
```diff
@@ -65,7 +65,7 @@ def output_gb_sas(
     # Map to the CORA statuses from the statusencoded column
     df = map_o.create_cora_status_col(df, cora_mapper)
 
-    # Map the sizebands based on frozen employment
+    # Map the sizebands based on  employment
```
Comment (JenCheshire) [inline src/outputs/gb_sas.py]: Again, small thing, but we've taken out frozen here - should we take it out from the function description in `map_output_cols.py` and where it's called in `outputs\short_form.py` and `outputs\tau.py`?

**Review by JenCheshire — DISMISSED** [review]
Look good - just a couple of (very) minor suggestions

File: src/outputs/long_form.py [hunk 4]
```diff
@@ -49,10 +49,10 @@ def output_long_form(
...
 
     # Create long form output dataframe with required columns from schema
-    schema_path = config["schema_paths"]["frozen_longform_schema"]
+    schema_path = config["schema_paths"]["longform_schema"]
     schema_dict = load_schema(schema_path)
     longform_output = create_output_df(df, schema_dict)
 
     tdate = datetime.now().strftime("%Y-%m-%d")
-    filename = f"Frozen_Base_Data_{tdate}_v{run_id}.csv"
+    filename = f"Base_Data_{tdate}_v{run_id}.csv"
```
Comment (JenCheshire) [inline src/outputs/long_form.py]: Off the back of our meeting - should this be called base_data or long_form?

File: src/outputs/gb_sas.py (hunk 3 above)
Comment (jwestw) [reply src/outputs/gb_sas.py]: I couldn't find what you were referring to here. But I found another mention of "frozen" in `map_output_cols.py`. Could you clarify where this is?

Comment (zorge69) [reply src/outputs/gb_sas.py]: In map_output_cols, the frozen employment is correct, no need to change.

Comment (zorge69) [reply src/outputs/gb_sas.py]: Please change the comment in tau.py, line 67: # # Map the sizebands based on column "employment"

File: src/outputs/long_form.py (hunk 4 above)
Comment (jwestw) [reply src/outputs/long_form.py]: I've changed this now.

File: src/pipeline.py (hunk 2 above)
Comment (jwestw) [reply src/pipeline.py]: I think I might have just added this in for consistency in the comments.

**Comment by jwestw** [comment]
@AnneONS @JenCheshire @zorge69 I think everything is resolved now.

**Review by JenCheshire — COMMENTED** [review]
Looks good to me

File: src/pipeline.py (hunk 2 above)
Comment (JenCheshire) [reply src/pipeline.py]: NI staging is being run twice in `pipeline.py` - a temp run here at line 90-106 (just staging) and again at line 142-145 (full staging and construction).  Should this bit be taken out now?

File: src/developer_config.yaml [hunk 5]
```diff
@@ -4,8 +4,8 @@ global:
   logging_level: "DEBUG"
   table_config: "SingleLine"
   # Environment settings
-  dev_test : False
-  network_or_hdfs: network #whether to load from hdfs or network (local Python)
+  dev_test : True
```
Comment (JenCheshire) [inline src/developer_config.yaml]: Change back to network

**Review by JenCheshire — COMMENTED** [review]
One question re: doubling up NI staging in `pipeline.py`  (see below), and a change to the config - otherwise all looking good

**Review by AnneONS — APPROVED** [review]
This is now ready to merge. I've tested running it.

**Pull request merged**